migtests/scripts/run-validate-assessment-report.sh

```diff
@@ -97,10 +97,12 @@ main() {
         actual_json_file="${EXPORT_DIR}/assessment/reports/migration_assessment_report.json"
 	    compare_json_reports ${expected_json_file} ${actual_json_file}
 
-		echo "Comparing HTML report"
-		expected_html_file="${TEST_DIR}/expectedAssessmentReport.html"
-		actual_html_file="${EXPORT_DIR}/assessment/reports/migration_assessment_report.html"
-		${SCRIPTS}/compare-html-reports.py ${expected_html_file} ${actual_html_file}
+		if [[ "${SKIP_HTML_VALIDATION}" != "true" ]]; then 
```

Comment:
what is the context?

Reply:
this is to avoid an overhead of updating various expected HTML files as we add new tests. Added a way to skip that check if not required